devicon.json

```
@@ -939,11 +957,10 @@
                 "plain-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "plain"
```

```suggestion
                "plain",
                "plain-wordmark"
```

Why you suggest to remove `plain-wordmark`? We have this icon: https://github.com/devicons/devicon/tree/develop/icons/foundation